KmsClient.js

```diff
@@ -76,26 +76,33 @@ export class KmsClient {
       capability = _getRootZcapId({keystoreId});
     }
 
+    let headers;
     try {
       // sign HTTP header
-      const headers = await signCapabilityInvocation({
+      headers = await signCapabilityInvocation({
         url, method: 'post', headers: this.defaultHeaders,
         json: operation, capability, invocationSigner,
         capabilityAction: 'generateKey'
       });
+    } catch(e) {
+      _handleClientError({
+        message: 'Error signing capability invocation while generating key',
```

Comment:
I believe there are a number of `message` entries that are missing punctuation, I'll just not this one.  I would expect the `_handlleClientError` helper to expect consistency wrt punctuation on the error message.

```suggestion
        message: 'Error signing capability invocation while generating key.',
```

Reply:
@mattcollier - this client is a bit of a unique case, where `_handleClientError` ensures the period at the end, over at https://github.com/digitalbazaar/webkms-client/pull/32/files#diff-47f7dfced7ea4bee3dc9780295b471b8212cc8b5bb2c852dc7b0fb1daa74ca5dR817
(I should remove the periods from the other error message params, tho, for consistency)

Reply:
Well, I think you're creating an API that is going to be difficult for people to understand given that it is not consistent with how we handle error messages everywhere else in the stack.  I think the proper solution to the situation you're trying to address is to strip periods in the *complete sentence* error messages passed into the API and replace them as needed.

Reply:
@mattcollier - ah, good call, ok

Reply:
@mattcollier - fixed in https://github.com/digitalbazaar/webkms-client/pull/32/commits/0600d8c44af74f89d388d9e5f7392b5ab1a86aac (added trailing periods for consistency).